x-pack/legacy/plugins/siem/public/containers/users/index.tsx

```diff
@@ -53,7 +54,7 @@ export interface UsersComponentReduxProps {
   sort: UsersSortField;
 }
 
-type UsersProps = OwnProps & UsersComponentReduxProps;
+type UsersProps = OwnProps & UsersComponentReduxProps & WithKibanaProps;
```

Comment:
That's an interesting type `& WithKibanaProps;` That's nice we have more access to Kibana Props now 👍